sprucebot-skill-kit.code-workspace

```diff
@@ -74,7 +74,7 @@
 		"eslint.options": {
 			"extensions": [".js", ".jsx", ".ts", ".tsx"]
 		},
-		"typescript.tsdk": "node_modules/typescript/lib",
+		"typescript.tsdk": "Workspace/node_modules/typescript/lib",
```

Comment:
I can't find this style documented for vscode anywhere; can you link me?

Reply:
Also all packages in the lerna monorepo should be using the same TS version, not sure why this would be needed?